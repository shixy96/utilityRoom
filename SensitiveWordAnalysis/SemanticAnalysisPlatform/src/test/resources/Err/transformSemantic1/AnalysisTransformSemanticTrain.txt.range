x
-1 1
1 0 12
2 0 14.75
3 0 24.75
4 0 17.5
5 0 14.25
6 0 16.5
7 0 24
8 0 27.75
9 0 24
10 0 27
11 0 18
12 0 26.5
13 0 16.5
14 0 12
15 0 11.5
16 0 16
17 0 10
18 0 10.5
19 0 10
20 0 3
21 0 3
