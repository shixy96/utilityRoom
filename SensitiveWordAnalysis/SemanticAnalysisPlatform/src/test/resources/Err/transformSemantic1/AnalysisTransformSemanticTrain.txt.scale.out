log2c=5 log2g=-7 rate=82.1849
log2c=-1 log2g=-7 rate=79.807
log2c=5 log2g=-1 rate=86.038
log2c=-1 log2g=-1 rate=83.1871
log2c=11 log2g=-7 rate=82.2907
log2c=11 log2g=-1 rate=87.5941
log2c=5 log2g=-13 rate=79.6825
log2c=-1 log2g=-13 rate=74.5969
log2c=11 log2g=-13 rate=82.1786
log2c=-3 log2g=-7 rate=74.7837
log2c=-3 log2g=-1 rate=82.3156
log2c=-3 log2g=-13 rate=74.5969
log2c=5 log2g=1 rate=87.9614
log2c=-1 log2g=1 rate=86.511
log2c=11 log2g=1 rate=87.9427
log2c=-3 log2g=1 rate=85.2163
log2c=9 log2g=-7 rate=82.1164
log2c=9 log2g=-1 rate=87.6688
log2c=9 log2g=-13 rate=82.1475
log2c=9 log2g=1 rate=87.8743
log2c=5 log2g=-11 rate=82.3903
log2c=-1 log2g=-11 rate=74.5969
log2c=11 log2g=-11 rate=82.1662
log2c=-3 log2g=-11 rate=74.5969
log2c=9 log2g=-11 rate=82.1538
log2c=3 log2g=-7 rate=82.1413
log2c=3 log2g=-1 rate=85.484
log2c=3 log2g=-13 rate=74.7775
log2c=3 log2g=1 rate=88.0921
log2c=3 log2g=-11 rate=79.695
log2c=5 log2g=-5 rate=82.1538
log2c=-1 log2g=-5 rate=82.2409
log2c=11 log2g=-5 rate=84.1208
log2c=-3 log2g=-5 rate=79.9129
log2c=9 log2g=-5 rate=82.9879
log2c=3 log2g=-5 rate=82.1289
log2c=15 log2g=-7 rate=83.517
log2c=15 log2g=-1 rate=87.7996
log2c=15 log2g=-13 rate=82.2098
log2c=15 log2g=1 rate=87.7249
log2c=15 log2g=-11 rate=82.5397
log2c=15 log2g=-5 rate=84.6499
log2c=5 log2g=-15 rate=74.7775
log2c=-1 log2g=-15 rate=74.5969
log2c=11 log2g=-15 rate=82.16
log2c=-3 log2g=-15 rate=74.5969
log2c=9 log2g=-15 rate=82.3965
log2c=3 log2g=-15 rate=74.5969
log2c=15 log2g=-15 rate=82.3903
log2c=-5 log2g=-7 rate=74.5969
log2c=-5 log2g=-1 rate=82.16
log2c=-5 log2g=-13 rate=74.5969
log2c=-5 log2g=1 rate=83.131
log2c=-5 log2g=-11 rate=74.5969
log2c=-5 log2g=-5 rate=74.709
log2c=-5 log2g=-15 rate=74.5969
log2c=5 log2g=3 rate=88.5465
log2c=-1 log2g=3 rate=87.7249
log2c=11 log2g=3 rate=88.2602
log2c=-3 log2g=3 rate=86.038
log2c=9 log2g=3 rate=88.3411
log2c=3 log2g=3 rate=88.5528
log2c=15 log2g=3 rate=88.1606
log2c=-5 log2g=3 rate=84.7806
log2c=7 log2g=-7 rate=82.1413
log2c=7 log2g=-1 rate=87.1833
log2c=7 log2g=-13 rate=82.3965
log2c=7 log2g=1 rate=88.0548
log2c=7 log2g=-11 rate=82.1538
log2c=7 log2g=-5 rate=82.4276
log2c=7 log2g=-15 rate=79.6825
log2c=7 log2g=3 rate=88.4034
log2c=5 log2g=-9 rate=82.1164
log2c=-1 log2g=-9 rate=74.7899
log2c=11 log2g=-9 rate=82.1475
log2c=-3 log2g=-9 rate=74.5969
log2c=9 log2g=-9 rate=82.1973
log2c=3 log2g=-9 rate=82.3965
log2c=15 log2g=-9 rate=82.5023
log2c=-5 log2g=-9 rate=74.5969
log2c=7 log2g=-9 rate=82.1475
log2c=1 log2g=-7 rate=82.3841
log2c=1 log2g=-1 rate=84.3262
log2c=1 log2g=-13 rate=74.5969
log2c=1 log2g=1 rate=87.1833
log2c=1 log2g=-11 rate=74.7775
log2c=1 log2g=-5 rate=82.1786
log2c=1 log2g=-15 rate=74.5969
log2c=1 log2g=3 rate=88.5403
log2c=1 log2g=-9 rate=79.7137
log2c=5 log2g=-3 rate=83.8655
log2c=-1 log2g=-3 rate=82.1413
log2c=11 log2g=-3 rate=85.3035
log2c=-3 log2g=-3 rate=82.0728
log2c=9 log2g=-3 rate=84.681
log2c=3 log2g=-3 rate=82.6704
log2c=15 log2g=-3 rate=86.704
log2c=-5 log2g=-3 rate=80.7345
log2c=7 log2g=-3 rate=84.3386
log2c=1 log2g=-3 rate=82.2222
log2c=13 log2g=-7 rate=82.7077
log2c=13 log2g=-1 rate=87.5755
log2c=13 log2g=-13 rate=82.3405
log2c=13 log2g=1 rate=87.9178
log2c=13 log2g=-11 rate=82.328
log2c=13 log2g=-5 rate=84.3511
log2c=13 log2g=-15 rate=82.216
log2c=13 log2g=3 rate=88.1793
log2c=13 log2g=-9 rate=82.328
log2c=13 log2g=-3 rate=86.0317
